### DISCUSSION TITLE
-webkit-scrollbar

### DISCUSSION THREAD

**Discussion by tienlx97** [facebook/stylex#626] [Q&A]
I saw on facebook.com some component can define this: 

```js
 hideScrollbar: {
    MsOverflowStyle: "x1pq812k",
    scrollbarWidth: "x1rohswg",
    "::-webkit-scrollbar_display": "xfk6m8",
    "::-webkit-scrollbar_height": "x1yqm8si",
    "::-webkit-scrollbar_width": "xjx87ck",
    $$css: !0
},
```

My question is : how can I define `webkit-scrollbar`. Because I can not do this way:

```js
  hideScrollbar: {
    // TODO
    // MsOverflowStyle: "x1pq812k",
    scrollbarWidth: "none",
    "::-webkit-scrollbar": {
      display: "none",
      height: 0,
      width: 0,
    },
  }
```

**Comment by nmn** [comment]
The style shown above is supported by the compiler but the lint rule disallows it as it is considered a deprecated style property now.

Because of a limitation of ESLint, it is not possible to define errors and warnings within the same rule. I'm working on an overhaul to the eslint plugin to create two separate rules for errors and warnings.

Feel free to suppress the ESLint error on this in the mean time.
